test/integration/integration.test.ts

```diff
@@ -305,21 +316,30 @@ describe('Integration tests', (): void => {
       assert(fee.basisPoints === 200)
       assert(fee.asset === 'USD')
       assert(fee.scale === 2)
-      expect(payment.receiveAmount).toMatchObject({
+      expect(completedOutgoingPayment.receiveAmount).toMatchObject({
         assetCode: 'EUR',
         assetScale: 2,
         value: 500n
       })
-      expect(payment.debitAmount).toMatchObject({
+      expect(completedOutgoingPayment.debitAmount).toMatchObject({
         assetCode: 'USD',
         assetScale: 2,
         value: 668n
       })
-      expect(payment.sentAmount).toMatchObject({
+      expect(completedOutgoingPayment.sentAmount).toMatchObject({
         assetCode: 'USD',
         assetScale: 2,
         value: 550n
       })
+
+      const incomingPaymentId = receiver.id.split('/').slice(-1)[0]
+      const incomingPayment = await getIncomingPayment(incomingPaymentId)
+      expect(incomingPayment.receivedAmount).toMatchObject({
+        assetCode: 'EUR',
+        assetScale: 2,
+        value: 501n
```

Comment:
I understand that the receivedAmount may be greater than the sentAmount during cross currency transactions (i.e. you need 0.02 USD to reach a minimum of 0.01 EUR).

However, this still feels wrong to me:

original receiveAmount = 500
sentAmount = 550
receivedAmount = 501

with an exchange rate of 0.91 as tested above, we have 550 * 0.91 = 500.5, which is under the received amount of 501. This signifies somewhere in ILP pay there is a `Math.ceil` that ends up making money out of nothing. Are my assumptions correct in the fact that _this_ seems like a problem?

Reply:
I was about to say that this could be because the stream was closed too late but given that the packet size is huge, that can't be it. I guess we have a rounding problem somewhere.

Reply:
Unfourtunately will need to dive in deeper into ILP pay, since this is just the default behaviour. Will merge in for now, since it is the status quo at this point anyway.